src/interfaces/wallet.h

```diff
@@ -88,7 +88,7 @@ class Wallet
     virtual std::string getWalletName() = 0;
 
     // Get a new address.
-    virtual BResult<CTxDestination> getNewDestination(const OutputType type, const std::string label) = 0;
+    virtual util::Result<CTxDestination> getNewDestination(const OutputType type, const std::string label) = 0;
```

Comment:
While touching this line here and in the override function in `src/wallet/interfaces.cpp`
```suggestion
    virtual util::Result<CTxDestination> getNewDestination(const OutputType type, const std::string& label) = 0;
```

Reply:
Picked up in #25616.